### DISCUSSION TITLE
Older versions of packages

### DISCUSSION THREAD

**Discussion by dejan-cti** [alpinejs/alpine#3616] [1. Help]
Hi there! 

We are using older version of AlpineJS (v2.8.0) in some of our projects. I want to use some of AlpineJS plugins (trap/focus) but there is no time/budget to update version to the latest. Is there a way to install older version of that package? I've looked into some CDNs and the oldest one was 3.3 which is not working with my version.

Thanks!

**Comment by ekwoka** [comment]
They didn't exist.

V2 didn't have an adaptive plugin system.

Very little is actually breaking between v2 and v3. v3 just gives you a lot more options.

I think, off the top of my head, the only thing that would break would be if you use a dynamic `x-ref` like `:x-ref`.

Did you try just updating and seeing if anything actually breaks?
